pyproject.toml

```diff
@@ -64,6 +64,9 @@ pytest-xdist = "^2.2"
 radon = "^4.5"
 requests-mock = "^1.8"
 timeout-decorator = "^0.5"
+jsonpickle = "^2.0"
```

Comment:
Whoops, looks like I missed a spot. These are only used by `tests/performance_test.py`, which I didn't intend to get picked up by pytest. I can fix that after this is merged.